synth.py

```diff
@@ -61,4 +61,10 @@
 templated_files = common.py_library(cov_level=100)
 s.move(templated_files, excludes=["noxfile.py"])
 
+s.replace(
+    "docs/conf.py",
+    '{"members": True}',
```

Comment:
```suggestion
    r'\{"members": True\}',
```
The second argument is a regex, special characters need escaping.

Reply:
Hmm... it appears that it works as is and the escape might not be needed, interesting.

Or I might have messed up something locally when testing the PR again...

Reply:
Yup, both works fine for me so i have changed to regex as you recommend.

Reply:
Sounds safer, yes.

I can confirm that the replacement works, please just commit the changes in the `docs/conf.py` file, too, otherwise the PR on its own will have no effect on the docs. :wink: